apps/ledger-live-mobile/src/newArch/features/Web3Hub/screens/Web3HubApp/components/Header/index.tsx

```diff
@@ -60,7 +64,8 @@ export default function Web3HubAppHeader({
               </>
             )}
           </Flex>
-          <TabButton count={2} navigation={navigation} />
+
+          <TabButton count={2} navigation={navigation} onclick={() => captureTab(manifest)} />
```

Comment:
`count` is still hard coded here and we should catch that in the test

it will probably be much easier to do with jotai tho